### PR TITLE
additional secredkey functionally and two databases added

### PR DESCRIPTION
**Pull request by Teodor-Maeli**
Mitko,, please review, I will add additional functionally.

### CODE REVIEW

File: src/main/java/com/example/studentmanagmentrest/repository/StudentRepository.java [hunk 1]
```diff
@@ -17,6 +18,8 @@
 @Repository
 public interface StudentRepository extends JpaRepository<Student, Long> {
 
+    @Autowired
```
Comment (DimitarRachev) [inline src/main/java/com/example/studentmanagmentrest/repository/StudentRepository.java:21]: Leftover For deletion?

**Review by DimitarRachev — APPROVED** [review]
Nice

File: src/main/java/com/example/studentmanagmentrest/repository/StudentRepository.java (hunk 1 above)
Comment (Teodor-Maeli) [reply src/main/java/com/example/studentmanagmentrest/repository/StudentRepository.java:21]: Done

File: src/main/java/com/example/studentmanagmentrest/key/service/SecretKeyServiceImpl.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+import java.util.Random;
+
+@Service
+@AllArgsConstructor
+public class SecretKeyServiceImpl implements CommandLineRunner {
+
+    KeyRepository repository;
+
+
```
Comment (ValioG) [inline src/main/java/com/example/studentmanagmentrest/key/service/SecretKeyServiceImpl.java:18]: Other simple implementation
////////////
private String randomGenerator() {
        return RandomStringUtils.randomAlphanumeric(40);
    }
//////////
Gradle dependency  
	implementation group: 'org.apache.commons', name: 'commons-lang3', version: '3.12.0'

File: src/main/java/com/example/studentmanagmentrest/key/model/SecretKey.java [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+import javax.persistence.*;
+
+@Entity
+@AllArgsConstructor
+@NoArgsConstructor
+@Setter
+@Getter
+public class SecretKey {
+
```
Comment (ValioG) [inline src/main/java/com/example/studentmanagmentrest/key/model/SecretKey.java:17]: Suggest using sequences for postgres DB
/////////
    @Id
    @SequenceGenerator(
            name = "secretKey",
            sequenceName = "secretKey_sequence",
            allocationSize = 1
    )
    @GeneratedValue(
            strategy = GenerationType.SEQUENCE,
            generator = "secretKey_sequence"
    )
    @Column(nullable = false,updatable = false)
    Long id;

**Review by ValioG — APPROVED** [review]

**Pull request merged**